mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -3046,10 +3026,10 @@ func resourceContainerClusterUpdate(d *schema.ResourceData, meta interface{}) er
 		log.Printf("[INFO] GKE cluster %s's master global access config has been updated to %v", d.Id(), config)
 	}
 
-    if d.HasChange("binary_authorization") {
+	if d.HasChange("binary_authorization") || d.HasChange("enable_binary_authorization") {
 		req := &container.UpdateClusterRequest{
 			Update: &container.ClusterUpdate{
-				DesiredBinaryAuthorization: expandBinaryAuthorization(d.Get("binary_authorization")),
+				DesiredBinaryAuthorization: expandBinaryAuthorization(d.Get("binary_authorization"), d.Get("enable_binary_authorization").(bool)),
```

Comment:
Looks like the panic is:

```
panic: interface conversion: interface {} is nil, not bool

goroutine 111826 [running]:
github.com/hashicorp/terraform-provider-google-beta/google-beta/services/container.resourceContainerClusterUpdate(0xc00b106880, {0x3fbfea0?, 0xc00f8f7500})
	/go/src/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/container/resource_container_cluster.go:2985 +0xa7c5
```

Here's a link to the diff in the downstream code used for running the tests:
https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-9629-old..auto-pr-9629#diff-3aaa21fdf4c9e6375dfe66f8142650926625edb14fd52ec4c8cdb03e1b76ee58R2985

Line 2985 there corresponds to this line here.

Reply:
Thanks! I'm not used to go, so I was greping for fail/error instead of panic.